src/wp-includes/rest-api/endpoints/class-wp-rest-attachments-controller.php

```diff
@@ -1014,4 +1238,44 @@ protected function check_upload_size( $file ) {
 		return true;
 	}
 
+	/**
+	 * Gets the request args for the edit item route.
+	 *
+	 * @since 5.5.0
+	 *
+	 * @return array
+	 */
+	protected function get_edit_media_item_args() {
+		return array(
+			'rotation' => array(
+				'description' => __( 'The amount to rotate the image clockwise in degrees.' ),
+				'type'        => 'integer',
+			),
+			'x'        => array(
+				'description' => __( 'As a percentage of the original image, the x position to start the crop from.' ),
```

Comment:
Just a nitpick :) Maybe drop `of the original image` as it's technically not needed. Percentage values work for any sub-size of the image. They come from the (scaled) image that is being edited. Perhaps something like `As a percentage of the  image, the x position to start the crop from.` or `The x position to start the crop from, in percents.`

Reply:
Ah I see, will update.

Reply:
Fixed.